### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RyanLi1028**
#### What is included in this PR?
updated readme.md to provide some more alternative options to set up and run the web UI locally, and addressed the issue that ODK hamster service is now outdated/deprecated.

### CODE REVIEW

**Comment by RyanLi0802** [comment]
#### What is included in this PR?
updated readme.md to provide some more alternative options to set up and run the web UI locally, and addressed the issue that ODK hamster service is now outdated/deprecated.

File: README.md [hunk 1]
```diff
@@ -18,15 +18,23 @@ Engage with the community and get technical support on [the ODK-X forum](https:/
...
+alternatively, if you've cloned the repository to your local computer, nagivate to the directory containing the DockerFile, and use
+
+`docker build -t odk/sync-web-ui .`
+
 ## Run
 
-Use [RUN.md](RUN.md).
+*Note that the ODK Hamster Service is outdated and may no longer be supported, you are encouraged to set up your local ODK sync-endpoint with [sync-endpoint-default-setup](https://github.com/opendatakit/sync-endpoint-default-setup) (to customize your local setup, please look into ``docker-compose.yml`` inside the ``sync-endpoint-default-setup`` repository for more details).*
+
+Option 1: Use [RUN.md](RUN.md).
```
Comment (wbrunette) [inline README.md]: It would be better to get rid of Run.md and bring all the instructions into the readme to be consistent with the other repos.

**Review by wbrunette — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RyanLi1028) [reply README.md]: should we keep the (outdated) instructions from RUN.MD in ReadMe, or should we simply delete everything in RUN.MD?

Comment (wbrunette) [reply README.md]: delete everything in run.md and and consolidate to a single instruction set in readme

File: README.md [hunk 2]
```diff
@@ -18,15 +18,27 @@ Engage with the community and get technical support on [the ODK-X forum](https:/
...
+
 ## Run
 
-Use [RUN.md](RUN.md).
+The Web UI must run in conjunction with ODK sync-endpoint. 
+The fastest way to get sync-endpoint set up is through following the build and run instructions of
+[sync-endpoint-default-setup](https://github.com/opendatakit/sync-endpoint-default-setup).
+
+Once everything is set and ready, in the `sync-endpoint-default-setup` directory, run command
+`docker stack deploy -c docker-compose.yml syncldap` to deploy all services
```
Comment (wbrunette) [inline README.md:36]: You should add some comments about the networking issues HTTP for local (likely) as HTTPS needs a certificate which you can't get for localhost.

**Review by wbrunette — CHANGES_REQUESTED** [review]

**Pull request merged**